src/validate.rs

```diff
@@ -751,6 +751,13 @@ pub fn validate_vic3_modifiers(vd: &mut Validator, sc: &mut ScopeContext) {
     });
 }
 
+#[cfg(feature = "imperator")]
+pub fn validate_imperator_modifiers(vd: &mut Validator, sc: &mut ScopeContext) {
```

Comment:
This can probably be combined with validate_vic3_modifiers if they really both use the same modifier format